sample-notificator-app/src/main/groovy/life/qbic/samplenotificator/DependencyManager.groovy

```diff
@@ -53,13 +55,20 @@ class DependencyManager {
         return properties
     }
 
-    private void setupCreateNotification(){
-        CreateNotificationDbConnector connector = new CreateNotificationDbConnector(DatabaseSession.getInstance())
-        CreateNotificationOutput someOutput = null //todo implement me
-        createNotification = new CreateNotification(connector,someOutput)
+    private void setupFetchSubscriber(){
+        FetchSubscriberDbConnector connector = new FetchSubscriberDbConnector(DatabaseSession.getInstance())
```

Comment:
Since the DS is called FetchSubscriberDbConnector I would actually not expect it to be used by some other use case. If another use case is implemented I expect the use case to have its own DB interface with individual interface methods (at least this is to what we agreed so far)